tests/codebuild/update_tests.sh

```diff
@@ -9,6 +9,12 @@ source inject_tests.sh
 function run_update_canary_tests
 {
   local crd_namespace="$1"
+
+  echo "Injecting variables into tests"
```

Comment:
I think we should revert changes to this file when we revert #155 so that we have this test run through integ before canary.

just being cautious for canaries here

Reply:
Ok, let me revert PR 155 as well and fix the underlying issue.

Reply:
Done